.ci/scripts/build-environment/downstream-builder/generate_downstream.sh

```diff
@@ -120,16 +120,12 @@ if [ "$REPO" == "terraform-google-conversion" ]; then
 
     bundle exec compiler.rb -a -e terraform -f validator -o $LOCAL_PATH/tfplan2cai -v $VERSION
 
-    # Ignore 'tgc-base' step, because this is a new provider.
-    # TODO: remove this condition on next cai2hcl-related PR.
```

Comment:
It looks to me like this could be removed in a separate PR. I think it would be better to do it in a standalone PR _before_ this PR so that this PR will show an accurate diff.

Reply:
Sure, here is a separate PR for this line change: https://github.com/GoogleCloudPlatform/magic-modules/pull/9407.

Will update this PR once merged.

Reply:
Done